mlir/include/mlir/Dialect/Rock/IR/RockOps.td

```diff
@@ -1046,8 +1046,12 @@ defvar LdsBufferTypes = GemmInputTypes # [VectorOfRankAndType<[1], GemmInputType
 def Rock_BlockwiseGemmOp:
     Rock_Op<"blockwise_gemm">,
     Arguments<(ins MemRefRankOf<GemmInputTypes, [3]>:$matrixA,
+                   BoolAttr : $isKContiguousDimA,
                    MemRefRankOf<GemmInputTypes, [3]>:$matrixB,
+                   BoolAttr : $isKContiguousDimB,
```

Comment:
In the operator? The point is that we will use that attribute to rotate and/or swap. If we name the operator property `rotateKwithD`, during the lowering we will have:
```
bool rotateDWithK = op.getRotateKWithD
bool doSwapThreadIterSubDimsForM = !rotateDWithK
```
If you wish, we might add a `doSwapThreadIterSubDimsForM` additional property to the blockwise gemm operator to keep things separated. What do you think?

Reply:
as discussed offline, I think they are not mutually exclusive from blockwiseGemm PoV.
i.e. it will still result in a valid kernel that is computing accurate output.

They become mutually exclusive in gridwise to blockwise lowering; so lets hoist the decision to the lowering.

Reply:
That was my bad, because we don't use the swap flag in blockwise, but we only use the rotation flag. So I added the two rotateMWithK and rotateNWithK